lib/client-assets.php

```diff
@@ -799,6 +799,41 @@ function gutenberg_capture_code_editor_settings( $settings ) {
 	$gutenberg_captured_code_editor_settings = $settings;
 	return false;
 }
+/**
+ * Retrieve a stored autosave that is newer than the post save.
+ *
+ * Deletes autosaves that are older than the post save.
```

Comment:
Not true. Deleted identical autosaves only when they are newer than the post (yep, kind of pointless) :)

Reply:
Will adjust docblock